charts/fluent-bit/values.yaml

```diff
@@ -93,6 +93,11 @@ extraVolumes: []
 
 extraVolumeMounts: []
 
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 1
```

Comment:
```suggestion
updateStrategy: {}
  # type: RollingUpdate
  # rollingUpdate:
  # maxUnavailable: 1
```

One last thing, let's leave this off by default.

Reply:
🙆‍♂️